client/src/App.test.jsx

```diff
@@ -165,4 +169,16 @@ test('Selecting the selected check-in or check-out date should pull up the calen
 
 
 
-})
\ No newline at end of file
+})
+
+
+
+// test('Shows calendar if URL has appropriate hash', async() => {
+//   var history = createMemoryHistory()
+
+//   history.push('#availability-calendar')
+//   render(<Router history={history}><App /></Router>)
+//   const calendar = await screen.findAllByTestId('calendar');
+//   expect(calendar[0].style._values.display).toEqual('block');
+
+// })
```

Comment:
Also think about writing some negative tests. I also need to get better at this 😬